### DISCUSSION TITLE
Version 1.0!

### DISCUSSION THREAD

**Discussion by rnd-ash** [rnd-ash/OpenVehicleDiag#20] [General]
# Version 1.0!

*NOTE*: Mac Users  - I could not get a pre-compiled binary for Mac Users so you will have to compile yourself!

<hr /><em>This discussion was created from the release <a href='https://github.com/rnd-ash/OpenVehicleDiag/releases/tag/v1.0.0'>Version 1.0!</a>.</em>

**Comment by exekutive** [comment]
if anybody has done this successfully, please share your results.
